spec/LandingPageBreastCancer.html

```diff
@@ -14,12 +14,12 @@ <h2>Table of Contents</h2>
 </ul>
 <h2><a name="Guidance"></a>Guidance for HL7 Voters</h2>
 <p>This section provides orientation to the ballot materials.</p>
-<h3>Where to Focus</h3>
+<h3>Where to SubjectOfInformation</h3>
```

Comment:
Change back to "Focus" in the landing page..  Looks like a typo from a global replace of the element "Focus" to "SubjectOfInformation" in data element files.